docs/basics/101-102-populate.rst

```diff
@@ -156,8 +156,8 @@ Cool, so now you have added some files to your dataset history. But what is a bi
 inconvenient is that both books were saved *together*. You begin to wonder: "A Python
 book and a Unix book do not have that much in common. I probably should not save them
 in the same commit. And ... what happens if I have files I don't want to track?
-:command:`datalad save -m "some commit message"` would write all of what is currently
-in my dataset and untracked or modified into the history!"
+:command:`datalad save -m "some commit message"` would save all of what is currently
+in my dataset untracked or modified into the history!"
```

Comment:
I agree with the change, I just find this sentence a bit bumpy like this because it does something weird with the sentence structure. What do you think of:

```suggestion
untracked or modified in my dataset into the history!"
```
?

Reply:
Yeah better, may be even replace my with the